pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -1663,6 +1663,7 @@
   hoc_faq_keep_learning_a_markdown: "Anyone can host an Hour of Code at any time. The tutorials stay up year-round. You can expect all our tutorials and curriculum to be available on our site in perpetuity. Please go to [our resources](%{promote}) for event how-to guides and other resources to help make your Hour of Code event a success."
   hoc_faq_certificates_q: "Where can I print certificates for my students?"
   hoc_faq_certificates_a_markdown: "Go to our [certificates page](%{certificates_url}/certificates) where you can print certificates for your entire class ahead of time."
```

Comment:
Is this string still being used anywhere after this change? If not, let's remove it as part of this PR!

Reply:
Removed ❌